Common/src/Pollster/TaskHandler.cs

```diff
@@ -1015,7 +1015,7 @@ await agentHandler_.Stop(lateCts_.Token)
                   };
       await HandleErrorResubmitAsync(e,
                                      taskData_,
-                                     earlyCts_.Token)
+                                     lateCts_.Token)
```

Comment:
I fail to see how this change relates to the PR description

Reply:
To check if the cancellation has been triggered, we need to check the proper cancellation token. During task execution, we use the late cancellation token, so we need to check the same.